### PR TITLE
Tominthe toon patch 2

### PR DESCRIPTION
**Pull request by TomintheToon**
Added new 'Program Addresses' page and page reference in summary.

### CODE REVIEW

**Comment by TomintheToon** [comment]
@DonDuala new PR with updated base. Hope this isn't causing you more hassle than worth!

**Comment by jlkramer16** [comment]
@brittcyr are we missing any relevant PDAs / should we include all of these?

File: extra/program-addresses.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+description: A list of Dual Finance Solana program addresses
+---
+
+# Program Addresses
+
+|Dual Finance Program| Solana Program Address|
+|---| ---|
+|DIP Smart Contract| DiPbvUUJkDhV9jFtQsDFnMEMRJyjW5iS6NMwoySiW8ki|
```
Comment (brittcyr) [inline extra/program-addresses.md]: These are all programs

File: extra/program-addresses.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+|SO Smart Contract | 4yx1NJ4Vqf2zT1oVLk4SySBhhDJXmXFt88ncm4gPxtL7|
+|DIP MM Smart Contract| esdUULhoeFHro46mvkTizMzg38hKBunvErmPuUw6n9p|
```
Comment (brittcyr) [inline extra/program-addresses.md]: I wouldnt include this one since it is not really user facing

Comment (TomintheToon) [reply extra/program-addresses.md]: Removed.

File: extra/program-addresses.md (hunk 1 above)
Comment (TomintheToon) [reply extra/program-addresses.md]: Updated labelling.

**Review by jlkramer16 — APPROVED** [review]

**Pull request merged**